### PR TITLE
Master theme issues var

### PR DESCRIPTION
**Pull request by var-odoo**
Related task: https://www.odoo.com/web#id=1938637&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
Pad: https://pad.odoo.com/p/r.800ba7770c62ede5fd8eb79362d031fa

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/event/controllers/main.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class EventController(Controller):
 
     @route(['''/event/<model("event.event", "[('state', 'in', ('confirm', 'done'))]"):event>/ics'''], type='http', auth="public")
-    def event_ics_file(self, event):
+    def event_ics_file(self, event, **kwargs):
```
Comment (qsm-odoo) [inline addons/event/controllers/main.py]: For saas-12.2

File: addons/website/static/src/scss/website.scss [hunk 2]
```diff
@@ -1050,6 +1050,12 @@ table.table_desc tr td {
     background: $light;
     margin-top: -999px;
     transition: margin-top 500ms ease 0s;
+    @include media-breakpoint-down(md) {
```
Comment (qsm-odoo) [inline addons/website/static/src/scss/website.scss]: To check but I really it is sm

**Comment by qsm-odoo** [comment]
@robodoo r-

**Comment by dja-odoo** [comment]
@qsm-odoo   I've changed the commit messages :) Could you have a look ? Thanks :)

**Pull request closed without merging**